packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -562,6 +566,7 @@ export class InputTimePicker
           aria-label={getLabelText(this)}
           class={{
             [CSS.container]: true,
+            [CSS.invertFocusTextColor]: userAgentPlatform == "Win32",
```

Comment:
I don't think we should do this. We have no idea what the selection color on a windows computer will be so flipping the text color isn't going to work if the OS is themed.

Reply:
Thanks to your help, this is now a 1-liner!